csrc/ir/interface_nodes.h

```diff
@@ -409,6 +409,13 @@ class NVF_API TensorView : public Val {
   void applyMmaSwizzle(MmaOperand operand);
   void applyMmaSwizzle(MmaInputSmemSwizzle swizzle);
 
+  //! Transforms the innermost iterdomains according to the given mma swizzle,
+  //!  this should be used on the tvs that are inputs of a MmaOp and are loaded
+  //!  using TMA.
+  void applyMmaSwizzleForTMALoad(
```

Comment:
Could we also unify this function with this?
https://github.com/NVIDIA/Fuser/blob/4478853dd8001e00b99e5c4f489f3d06e4fecd06/tests/cpp/test_memory.cpp#L401-L413

Reply:
Please take another look